### ISSUE TITLE
[jss-starter-kit] imports are leaking into bundle

### ISSUE THREAD

**Issue by appsforartists** [cssinjs/jss#1032]
I just tried to fix #948, but I can't because rollup isn't inlining the bundles correctly anymore:

```javascript
import warning from 'warning';
import functions from 'jss-plugin-syntax-rule-value-function';
import observable from 'jss-plugin-syntax-rule-value-observable';
import template from 'jss-plugin-syntax-template';
import global$1 from 'jss-plugin-syntax-global';
import extend from 'jss-plugin-syntax-extend';
import nested from 'jss-plugin-syntax-nested';
import compose from 'jss-plugin-syntax-compose';
import camelCase from 'jss-plugin-syntax-camel-case';
import defaultUnit from 'jss-plugin-syntax-default-unit';
import expand from 'jss-plugin-syntax-expand';
export { default as functions } from 'jss-plugin-rule-value-function';
export { default as observable } from 'jss-plugin-rule-value-observable';
export { default as template } from 'jss-plugin-template';
export { default as global } from 'jss-plugin-global';
export { default as extend } from 'jss-plugin-extend';
export { default as nested } from 'jss-plugin-nested';
export { default as compose } from 'jss-plugin-compose';
export { default as camelCase } from 'jss-plugin-camel-case';
export { default as defaultUnit } from 'jss-plugin-default-unit';
export { default as expand } from 'jss-plugin-expand';
```

I'm guessing this is related to one of the recent rollup version/build script changes, but I haven't bisected to be certain.

**Comment by HenriBeck** [comment]
The starter kit still looks good for me. Also, this issue was fixed in #1001

**Comment by appsforartists** [comment]
Weeeird.  I just ran `git bisect` twice.  The first time, every revision had this problem.  The second time, none of them did.

This should all be hermetic within yarn.lock, so I'm not sure why I'm not getting stable results.  Since you aren't seeing the problem now, and I am not anymore, I'll close this.
